macos/generate_wazuh_packages.sh

```diff
@@ -75,6 +78,7 @@ function help() {
     echo "    -h, --help                [  Util  ] Show this help."
     echo "    -i, --install-deps        [  Util  ] Install build dependencies (Packages)."
     echo "    -x, --install-xcode       [  Util  ] Install X-Code and brew. Can't be executed as root."
+    echo "    -k, --checksum            [Optional] Generate checksum"
```

Comment:
Add this option before the help and util options.

Reply:
Right :D https://github.com/wazuh/wazuh-packages/pull/192/commits/f7a3b51c066ee747b8cbe265e3cf0713b6c6427f